### PR TITLE
app-dicts/myspell-de: Version bump

### PR DESCRIPTION
**Pull request by thecb1**
* Current Version is 2017.01.12
* Added myself as proxied maintainer
* Set EAPI to 7 for old ebuild

Closes: https://bugs.gentoo.org/610548
Signed-off-by: Christian Buchinger <cb-gentoo@guya.de>
Package-Manager: Portage-2.3.51, Repoman-2.3.11

### CODE REVIEW

**Comment by thecb1** [comment]
It came to my knowledge that an EAPI change must imply a revbump, which would also include dropping all stable arches to ~. So I guess it would be better to keep myspell-de-2015.12.28.ebuild on EAPI 5 for now and remove it once the new version is stable on some arches?

**Comment by mgorny** [comment]
Yes, please don't change EAPI of old ebuilds. It's sufficient that the new one is using new EAPI.

File: app-dicts/myspell-de/myspell-de-2017.01.12.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+inherit myspell-r2
+
+DESCRIPTION="German (AT,CH,DE) dictionaries for myspell/hunspell"
+HOMEPAGE="
+	https://extensions.libreoffice.org/extensions/german-de-at-frami-dictionaries
+	https://extensions.libreoffice.org/extensions/german-de-ch-frami-dictionaries
+	https://extensions.libreoffice.org/extensions/german-de-de-frami-dictionaries
+"
+SRC_URI="
+	https://extensions.libreoffice.org/extensions/german-de-at-frami-dictionaries/$(ver_rs 1 -)/@@download/file/dict-de_AT-frami_$(ver_rs 1- -).oxt
```
Comment (mgorny) [inline app-dicts/myspell-de/myspell-de-2017.01.12.ebuild]: `$(ver_rs 1- -)` looks like a good candidate for putting into `MY_PV` to avoid repetition.

Comment (thecb1) [reply app-dicts/myspell-de/myspell-de-2017.01.12.ebuild]: Done. Also dropped IUSE and added default to src_prepare().

**Comment by mgorny** [comment]
One more request: please move the new version into summary line, so it's visible from `git log --oneline`.

**Pull request closed without merging**